cea/default.config

```diff
@@ -194,62 +194,61 @@ year.help = the year to calculate radiation for
 [solar]
 date-start = 2016-01-01
 date-start.type = DateParameter
-date-start.help =  site specific input (format: yyyy-mm-dd)
+date-start.help =  Starting date
 
-type-PVpanel = PV1
-type-PVpanel.type = ChoiceParameter
-type-PVpanel.choices = PV1 PV2 PV3
-type-PVpanel.help = type of panels, for PVT, please choose type_PVpanel = 'PV1', type_SCpanel = 'SC1',
-                    PV1: monocrystalline, PV2: poly, PV3: amorphous. please refer to supply system database.
+type-pvpanel = PV1
+type-pvpanel.type = ChoiceParameter
+type-pvpanel.choices = PV1 PV2 PV3
+type-pvpanel.help = Type of PV panel
 
-type-SCpanel = SC1
-type-SCpanel.type = ChoiceParameter
-type-SCpanel.choices = SC1 SC2
-type-SCpanel.help = type of solar collector, SC1: flat plat collectors, SC2: evacuated tubes
+type-scpanel = SC1
+type-scpanel.type = ChoiceParameter
+type-scpanel.choices = SC1 SC2
```

Comment:
@daren-thomas  I have a question on how can users add their customized technologies? For example, adding a `SC3` in the `region/economics/supply_systems.xls` database. 
Is there a way to go check on what is in the database, and then form the dropdown choices?

Reply:
Currently, no, we don't have a way to do that... we could think up a way, though by making a class called "ExcelLookupChoiceParameter" or something. Not quite sure if we need it. But go ahead and create an issue for this :-)